api/services/ocean/rightsizing/schemas/oceanRightsizingUpdateRuleRequest.yaml

```diff
@@ -0,0 +1,37 @@
+type: object
+title: Ocean Rightsizing Rule
+description: >
+  Determines the Ocean Rightsizing rule
+required:
+  - recommendationApplicationIntervals
+properties:
+  restartPods:
```

Comment:
restartPods field is obsolete. Should be removed.

Reply:
fixed